draft-ietf-quic-multipath.md

```diff
@@ -376,10 +374,11 @@ Reset ({{Section 10.3 of QUIC-TRANSPORT}}) closes the connection.
 
 ### Use PATH_ABANDON Frame to Close a Path {#path-abandon-close}
 
-Both endpoints, namely the client and the server, can close a path,
+Both endpoints, namely the client and the server, can initiate path closure,
 by sending PATH_ABANDON frame (see {{path-abandon-frame}}) which
-abandons the path with a corresponding Path Identifier. The PATH_ABANDON
-frame contains the Path Identifier and therefore can be sent on any path.
+which requests the peer to stop sending packets with the corresponding Destination Connection ID.
+The PATH_ABANDON frame contains the Destination Connection
+ID Sequence Number and therefore can be sent on any path.
```

Comment:
This is entirely my fault with PR #174 but this is actually specified already later in the text and therefore redundant here. However, we can also just leave this for now and make another editorial pass later.

Reply:
Okay, should we open an (editorial) issue, so we don't forget? Or we can create a new PR right after this one is merged (without an issue).

Reply:
Filled #194.